alembic/versions/591a56cee781_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 591a56cee781
+Revises: 0c90faa0378f, 64ed9566f189
+Create Date: 2024-09-25 14:56:42.211982
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '591a56cee781'
+down_revision: Union[str, None] = ('0c90faa0378f', '64ed9566f189')
```

Comment:
This is a merge migration, essentially when there are two heads, its hard for alebmic to determine the order in which migrations should be applied.  This kind of just states that these two can be applied in either order, then this merge migration becomes the new head.